framework/source/class/qx/data/SingleValueBinding.js

```diff
@@ -373,7 +373,7 @@ qx.Class.define("qx.data.SingleValueBinding",
         // if its the last property
         if (j == context.propertyNames.length - 1) {
           // if its an array
-          if (qx.Class.implementsInterface(source, qx.data.IListData)) {
+          if (qx.Class.implementsInterface(source, qx.data.IListData) && context.arrayIndexValues[i] !== "") {
```

Comment:
It looks like the variable should be `j`, not `i` - the linter says `i` is an unknown global symbol. I wonder why it works at all for you...

Reply:
Yes, of course it's j, I don't have the repo on the machine I'm on right now so edited the file directly from github and missed that, I'm updating the pr.